drivers/SmartThings/zigbee-thermostat/fingerprints.yml

```diff
@@ -85,10 +85,15 @@ zigbeeManufacturer:
     model: ST218
     deviceProfileName: thermostat-temperature-temperaturealarm
   - id: "Stelpro/MaestroStat"
-    deviceLabel: Stelpro Thermostat
+    deviceLabel: Maestro Thermostat
     manufacturer: Stelpro
-    model: MaestroStat
+    model: SMT402AD
```

Comment:
Hello @gcavalcantistelpro . I would recommend that we limit the scope of this PR 1727, to be specific to adding the new SMT402AD device. Could you please remove the changes to the device label, and model for this device?

Reply:
I would recommend if you want to proceed with making changes to that Masetro Thermostat device, that the changes needed on [be](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1499/files) reviewed, and instead move forward with that PR. 

Thank you. 

cc: @Brianj94

Reply:
Hello @lelandblue , do you mean to add SMT402AD model and keep the MaestroStat? so instead of replacing it, it will be a new model? keep in mind that the MaestroStat model does not work with this device, SMT402AD is the right model that makes the driver functionnal

Reply:
That's correct.